common/coin_support/wallet.h

```diff
@@ -42,13 +42,13 @@
 #define MAX_NUMBER_OF_MNEMONIC_WORDS 24
 
 /// Max length of mnemonic word
-#define MAX_MNEMONIC_WORD_LENGTH 16
+#define MAX_MNEMONIC_WORD_LENGTH 9
```

Comment:
need same size as the `ui_list` function parameter i.e. `MAX_UI_LIST_CHAR_LEN`.

Reply:
```suggestion
#define MAX_MNEMONIC_WORD_LENGTH MAX_UI_LIST_CHAR_LEN
```